frontend/src/components/general/file_upload/file_upload.svelte

```diff
@@ -1,12 +1,52 @@
 <script>
 	import ToggleList from '../form/toggle_list/toggle_list.svelte';
```

Comment:
Ignore this file, its just demoing some components right now till I finish filling it in later